tar/ccache/ccache_read.c

```diff
@@ -79,6 +79,11 @@ read_rec(void * cookie)
 	ccr->flags = 0;
 
 	/* Sanity check some fields. */
+	if (le64dec(ccre.nch) > (uint64_t)SIZE_MAX) {
+		warn0("Cache record has too many chunks for this platform;"
```

Comment:
never mind, I see it now.  My eyes focused on `        FILE * f;       /* File handle. */` and totally skipped over the `        char * s;       /* File name. */` right above it.  :(